README.md

```diff
@@ -273,129 +127,57 @@ product = shopify.Product()
 product.title = "Shopify Logo T-Shirt"
 product.id                          # => 292082188312
 product.save()                      # => True
-
 shopify.Product.exists(product.id)  # => True
-
 product = shopify.Product.find(292082188312)
 # Resource holding our newly created Product object
 # Inspect attributes with product.attributes
-
 product.price = 19.99
 product.save()                      # => True
 product.destroy()
 # Delete the resource from the remote server (i.e. Shopify)
 ```
 
-The [tests for this package](https://github.com/Shopify/shopify_python_api/tree/master/test) also serve to provide advanced examples of usage.
-
 ### Prefix options
 
-Some resources such as `Fulfillment` are prefixed by a parent resource in the Shopify API.
-
-e.g. `orders/450789469/fulfillments/255858046`
+Some resources such as `Fulfillment` are prefixed by a parent resource in the Shopify API (e.g. `orders/450789469/fulfillments/255858046`) In order to interact with these resources, you must specify the identifier of the parent resource in your request.
 
-In order to interact with these resources, you must specify the identifier of the parent resource in your request.
-
-e.g. `shopify.Fulfillment.find(255858046, order_id=450789469)`
+```python
+shopify.Fulfillment.find(255858046, order_id=450789469)
+```
 
 ### Console
-
 This package also includes the `shopify_api.py` script to make it easy to
 open up an interactive console to use the API with a shop.
```

Comment:
This package also includes the `shopify_api.py` script to make it easy to
open _~up~_ an interactive console to use the API with a shop.